web-app/js/smartR/d3Heatmap.js

```diff
@@ -336,7 +338,7 @@ SmartRHeatmap = (function(){
                     var rowValues = [];
                     for(var i = 0; i < significanceValues.length; i++) {
                         var significanceValue = significanceValues[i];
-                        rowValues.push([i, significanceValue]);
+                        rowValues.push([i, Math.abs(significanceValue)]);
```

Comment:
Why do you convert it here?
The value itself should remain negative as far as I know - we should just treat it as positive for sorting and bar length - if you take abs() here info about the sign will be lost and everything will be blue.